### PR TITLE
Fixes hat loadouts

### LINKED ISSUE #16870: Loadout Hat Errors. Loadouts spawning and giving non-existent hats
Reporting client version: 514.1585

## Round ID:
N/A

## Testmerges:
N/A

## Reproduction:

While setting up your loadout, under the hat options for wearing/usage a majority of the hats (primarily costume hats such) show up as errors. This applies also to when you spawn with them in game, and it should be noted that some (or all I have not checked) still work properly in game, they can be spawned and/or acquired naturally. I'm aware as of the posting of this that there was an upstream problem in regards to repathing. I wonder if this issue is related. I have compiled a complete list (I hope this is all of them) of the hats affected in the hat section of loadouts.
Mail Cap
Bandana (Pirate Bandana from Costume Vendor)
Xenos Helmet
Pirate Hat
Rice Hat
Nurse Hat (Locked/Unavailable)
TMC Hat
PG Hat
Deckers Hat
Morning Star Hat
Fancy Hat

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request

Fixes #16870

## How This Contributes To The Skyrat Roleplay Experience

Bugs bad

## Changelog

:cl:
fix: Hats in loadout work correctly againn
/:cl:

### CODE REVIEW

**Comment by Dreathtil** [comment]
Hopefully this particular item ends up alright as its currently one of the items in the loadout menu broked.

```
datum/loadout_item/head/bandana
	name = "Bandana"
	item_path = /obj/item/clothing/head/bandana
```
	
The item repathing PR has it listed differently now, as it is technically the Pirate bandana in code, just blankly listed as "Bandana" as all the rest of the bandanas are now under face masks
	
Item path might have to be changed to this
	
`/obj/item/clothing/head/costume/pirate/bandana`
	
Is what it is now in the costume vendor

**Comment by YakumoChen** [comment]
let me know if any other hats or items are popping up
there should have been a script done for this like in the suit PR but alas.

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm [hunk 1]
```diff
@@ -590,23 +590,23 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
...
 
 /datum/loadout_item/head/deckers
 	name = "Deckers Hat"
-	item_path = /obj/item/clothing/head/deckers
+	item_path = /obj/item/clothing/head/costume/deckers
 
 /datum/loadout_item/head/morningstar
 	name = "Morningstar Hat"
-	item_path = /obj/item/clothing/head/morningstar
+	item_path = /obj/item/clothing/head//costume/morningstar
```
Comment (tf-4) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm]:
```suggestion
	item_path = /obj/item/clothing/head/costume/morningstar
```

**Review by tf-4 — CHANGES_REQUESTED** [review]
just one mistype

**Review by GoldenAlpharex — APPROVED** [review]
There's only a few more fixes to do for hats, mostly ctrl-shift-F mass replace sort of stuff.

**Comment by YakumoChen** [comment]
I have a feeling it'd be easier to write a script than try this actually

**Comment by YakumoChen** [comment]
I'll maybe work on this later

**Pull request closed without merging**